DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -48,7 +49,10 @@ ICentresService centresService
                 !modelState.HasError(nameof(PersonalInformationViewModel.SecondaryEmail)) &&
                 model.SecondaryEmail != null;
 
-            bool? primaryEmailMatchesCentre = null;
+            var autoRegisterManagerEmail = isRegisterAdminJourney
+                ? centresDataService?.GetCentreAutoRegisterValues(model.Centre!.Value)

```

Comment:
I think the `centresDataService` is always non-null here (because of `isRegisterAdminJourney` being true) so we can switch null-coalescing operator `?.` to null-forgiving `!.` here.

Reply:
changed :)